fileset.go

```diff
@@ -39,16 +43,36 @@ type FileSet struct {
 // specified extension. Conversely, names without an extension will match all
 // files whose stem is the last path component of the filename.
 func Open(name string, paths ...fs.FS) *FileSet {
-	if filepath.IsAbs(name) {
-		panic("Open does not take absolute paths; use os.Open instead.")
-	}
-	if strings.HasPrefix(name, "."+string(filepath.Separator)) || strings.HasPrefix(name, ".."+string(filepath.Separator)) {
-		panic("Open does not take cwd-relative paths; use os.Open instead.")
+	checkNames(name)
+	if paths == nil {
+		paths = ConfigPaths()
 	}
+	return &FileSet{fs: paths, names: []string{name}}
+}
+
+// OpenMultiple opens a set of configuration files with one or more names.
+//
+// The order of the `names` slice defines a precedence of files across all
+// search paths. With a single `name`, the behaviour is identical to `Open`.
+//
+// The iteration order is described in detail in `Next()`.
+func OpenMultiple(names []string, paths []fs.FS) *FileSet {
```

Comment:
nit: I'd make `paths` variadic like Open(); almost all use-cases do not override the search path, and `OpenMultiple([]string{"foo", "bar"})` seems to be better working than  `OpenMultiple([]string{"foo", "bar"}, nil)` for that purpose.

I don't really feel strongly about this except maybe a slight preference towards the consistency between signatures.

Maybe this could also be a good time to rework the Open signature, too. But probably not for this PR.

I'm thinking this because something I want to eventually support is .d config dirs; presumably this would extend the file set to look at files in directories, but overall this seems more likely to be the thing that changes between applications (moreso than the search path, which is mostly standardized). So maybe, really, what we want is these functions:

    func Open(globs ...string) *FileSet
    func OpenWithOpts(globs []string, opts ...OpenOpt) *FileSet

Where the main usage of Open would be `Open("lvm/lvm.conf")` and `Open("yum.conf", "yum.d/*.conf")`, as an example. OpenWithOpts would only be there if the user has the less-conventional use-case of overriding the search path.